internal/scaffold/olm-catalog/csv_updaters.go

```diff
@@ -277,6 +279,11 @@ func (store *updaterStore) AddOwnedCRD(yamlDoc []byte) error {
 			Kind:    kind,
 		}
 		store.crds.crIDs[crdDescID(crdDesc)] = struct{}{}
+		// Parse CRD descriptors from source code comments and annotations.
+		gvk := schema.GroupVersionKind{Group: crd.Spec.Group, Version: ver, Kind: kind}
+		if err := descriptor.GetCRDDescriptorForGVK(scaffold.ApisDir, &crdDesc, gvk); err != nil {
+			return errors.Wrapf(err, "failed to set CRD descriptors for %s", gvk)
```

Comment:
Would it break anything if we used go 1.13's error wrapping?

```suggestion
			return fmt.Errorf("failed to set CRD descriptors for %s: %w", gvk, err)
```

Ditto elsewhere in PR

Reply:
`errors.Wrap{f}` [implements `Unwrap`](https://github.com/pkg/errors/blob/master/errors.go#L162), so it works with go 1.13 error wrapping.